### ISSUE TITLE
[GlobalStep] "Screen Reader" text overlaps with "Date Range" header text of "Date Filter" on "Analytics" pages.

### ISSUE THREAD

**Issue by gglobalstep** [woothemes/woocommerce#39793]
### Prerequisites

- [ ] I have carried out troubleshooting steps and I believe I have found a bug.
- [ ] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

"Screen Reader" text overlaps with "Date Range" header text of "Date Filter" on "Analytics" pages.

### Expected behavior

"Screen Reader" text  should be displayed properly without overlapping with "Date Range" header text of "Date Filter" on "Analytics" pages.

### Actual behavior

"Screen Reader" text overlaps with "Date Range" header text of "Date Filter" on "Analytics" pages.

### Steps to reproduce

1. Create any test site.
2. Install and activate all the required plugins.
3. Install an activate Woocommerce 8.1.0-dev plugin.4. 
4. Go to "Woocommerce > Settings > Advanced > Features"
5. Enable "New product editor".
6. Go to any "Analytics" pages.(i.e. Overview, revenue etc).
7. Click "Date range" filter dropdown.
8. Observe that, "Screen Reader" text overlaps with "Date Range" header text of "Date Filter".

### Video:

![#39793](https://github.com/woocommerce/woocommerce/assets/41110392/a0fdc574-3842-44c2-b0bd-eb9d8b4b90d7)

### WordPress Environment

Woocommerce Version: 8.1.0-dev
WordPress version: 6.3, 6.2
PHP version: 8.0.30

PC:
Windows 10
Chrome(Version 116.0.5845.97)
Firefox (Version 116.0.3)

### Isolating the problem

- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

<details>

`
### WordPress Environment ###

WC Version: 8.1.0
REST API Version: ✔ 8.1.0
WC Blocks Version: ✔ 10.6.5
Action Scheduler Version: ✔ 3.6.2
Log Directory Writable: ✔
WP Version: 6.3
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US
External object cache: –

### Server Environment ###

Server Info: Apache/2.4.57 (Unix) OpenSSL/1.0.2g
PHP Version: 8.0.30
PHP Post Max Size: 1 GB
PHP Time Limit: 30
PHP Max Input Vars: 5000
cURL Version: 7.47.0
OpenSSL/1.0.2g

SUHOSIN Installed: –
MySQL Version: 5.7.33-0ubuntu0.16.04.1-log
Max Upload Size: 512 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 8.1.0
WC Database Prefix: wp_
Total Database Size: 5.37MB
Database Data Size: 3.65MB
Database Index Size: 1.72MB
wp_woocommerce_sessions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_order_items: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_order_itemmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_actions: Data: 0.05MB + Index: 0.11MB + Engine InnoDB
wp_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_logs: Data: 0.05MB + Index: 0.03MB + Engine InnoDB
wp_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_comments: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_options: Data: 2.47MB + Index: 0.08MB + Engine InnoDB
wp_postmeta: Data: 0.06MB + Index: 0.03MB + Engine InnoDB
wp_posts: Data: 0.05MB + Index: 0.06MB + Engine InnoDB
wp_termmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_terms: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_term_relationships: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_term_taxonomy: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_usermeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_users: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_admin_notes: Data: 0.06MB + Index: 0.00MB + Engine InnoDB
wp_wc_admin_note_actions: Data: 0.05MB + Index: 0.02MB + Engine InnoDB
wp_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_customer_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_orders: Data: 0.02MB + Index: 0.11MB + Engine InnoDB
wp_wc_orders_meta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_addresses: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_operational_data: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_product_lookup: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_wc_order_stats: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_product_attributes_lookup: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_download_directories: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_meta_lookup: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_wc_rate_limits: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_reserved_stock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB

### Post Type Counts ###

attachment: 24
page: 7
post: 2
wp_navigation: 1
wp_template: 2

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (3) ###

Query Monitor: by John Blackbourn – 3.13.1
Companion Plugin: by Osk – 1.30
WooCommerce: by Automattic – 8.1.0-dev

### Inactive Plugins (3) ###

Akismet Anti-Spam: Spam Protection: by Automattic - Anti Spam Team – 5.2
Hello Dolly: by Matt Mullenweg – 1.7.2
Jetpack: by Automattic – 12.4

### Dropin Plugins (1) ###

db.php: Query Monitor Database Class (Drop-in)

### Settings ###

API Enabled: –
Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: –
Enforce Approved Product Download Directories: ✔
HPOS feature screen enabled: –
HPOS feature enabled: –
Order datastore: WC_Order_Data_Store_CPT
HPOS data sync enabled: –

### WC Pages ###

Shop base: #8 - /shop/
Cart: #9 - /cart/
Checkout: #10 - /checkout/
My account: #11 - /my-account/
Terms and conditions: ❌ Page not set

### Theme ###

Name: Twenty Twenty-Three
Version: 1.2
Author URL: https://wordpress.org
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: –

### Admin ###

Enabled Features: activity-panels
analytics
product-block-editor
coupons
core-profiler
customer-effort-score-tracks
import-products-task
experimental-fashion-sample-products
shipping-smart-defaults
shipping-setting-tour
homescreen
marketing
mobile-app-banner
navigation
onboarding
onboarding-tasks
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
shipping-label-banner
subscriptions
store-alerts
transient-notices
woo-mobile-welcome
wc-pay-promotion
wc-pay-welcome-page

Disabled Features: customize-store
minified-js
new-product-management-experience
product-variation-management
settings
async-product-editor-category-field

Daily Cron: ✔ Next scheduled: 2023-08-18 15:07:52 +00:00
Options: ✔
Notes: 61
Onboarding: skipped

### Action Scheduler ###

Complete: 80
Oldest: 2023-08-17 15:08:55 +0000
Newest: 2023-08-17 15:44:47 +0000

Failed: 1
Oldest: 2023-08-17 15:09:52 +0000
Newest: 2023-08-17 15:09:52 +0000

Pending: 1
Oldest: 2023-08-18 15:08:55 +0000
Newest: 2023-08-18 15:08:55 +0000

### Status report information ###

Generated at: 2023-08-17 15:56:04 +00:00
`

</details>

**Comment by tomalec** [comment]
@gglobalstep could you please provide more details on steps to reproduce and check if that's still an issue with the latest WooCommerce? I was not able to produce it.

[`screen-reader-text` is meant to be hidden, and visible for screen readers only](https://make.wordpress.org/accessibility/handbook/markup/the-css-class-screen-reader-text/). So the fact that is visible is the problem itself, regardless of where it's visible and what it covers ;)

**Comment by gglobalstep** [comment]
Hi @tomalec,

We are unable to reproduce the issue on our end using latest release of WooCommerce 9.7.1.

![Image](https://github.com/user-attachments/assets/5b1f0f63-3406-49e8-a27a-4bc81164080b)

**Comment by tomalec** [comment]
Excellent, thank you for checking. I'm closing the issue. Feel free to reopen if you experience it again.
